### PR TITLE
Version log for share

### PR DESCRIPTION
**Pull request by ulziibay**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+## [1.5.2] - 2018-09-04
+
+- Added sharing functionality with the following options.
```
Comment (lediur) [inline CHANGELOG.md]: Can you follow the `### Added` headings to be consistent with the entries below?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+## [1.5.2] - 2018-09-04
```
Comment (lediur) [inline CHANGELOG.md]: This isn't a bugfix so I would call this a minor version bump `1.6.0`

File: CHANGELOG.md (hunk 1 above)
Comment (ulziibay) [reply CHANGELOG.md]: Ok

File: CHANGELOG.md (hunk 2 above)
Comment (ulziibay) [reply CHANGELOG.md]: SG!

**Pull request merged**